crates/matrix-sdk-common/src/linked_chunk/mod.rs

```diff
@@ -933,7 +934,7 @@ impl ChunkIdentifierGenerator {
 /// Learn more with [`ChunkIdentifierGenerator`].
 #[derive(Copy, Clone, Debug, PartialEq)]
 #[repr(transparent)]
-pub struct ChunkIdentifier(u64);
+pub struct ChunkIdentifier(pub(super) u64);
```

Comment:
Interesting. Fwiw in the sqlite PR, I've introduced an `index()` method to retrieve this value, since I need it in many places; feel free to introduce that here as a new public method, and keep the inner `u64` private.